TROUBLESHOOTING.md

```diff
@@ -80,9 +80,13 @@ To read the error message returned by SendGrid's API:
   request.path = '/v3/mail/send'
   request.body = requestBody
   sg.API(request, function (error, response) {
-    console.log(response.statusCode)
-    console.log(response.body)
-    console.log(response.headers)
+    
```

Comment:
Good idea!
I have already committed the new code, which include the `else` part.
Thanks.